### ISSUE TITLE
What sequence length should I use for GLUE fine-tuning?

### ISSUE THREAD

**Issue by smr97** [google-research/bert#954]
Hi, I am trying to reproduce BERT-base accuracies on GLUE, and have a 32 GB v100 GPU. I wanted to know what should I set the `max_seq_length` to, in order to have a fair comparison with the results in the paper?

I dug into the appendix and found that you use 128 for 90% of the pretraining and 512 for 10%. The paper says that 
> most hyper params are same for fine-tuning with the exception of the batch size, learning rate, and number of training epochs.

Now I am not sure what this implies with respect to the sequence length. Could one of the authors please clarify this?

Thanks!
